main/templates/main/index.html

```diff
@@ -3,78 +3,59 @@
 {% block content %}
 
   <div class="container">
-    <div class="jumbotron">
-      <h1>Juno's <i>Personal Data Exploratory</i></h1>
-      <p>
-        📊📓 Personal Data Notebooks allow you to explore and analyze your personal data that is stored in <i>Open Humans</i>
-        right in your web browser. The <i>Exploratory</i> allows you to find and use interesting notebooks right away.
-      </p>
-  </div>
     <div class="row">
-      <div class="col-md-6">
-        <h2>Data Sources</h2>
+      <div class="col-xs-12">
         <p>
-          The <i>Exploratory</i> contains ready-to-use notebooks for many different data sources which are on Open Humans.
-          Click the buttons below to find notebooks that make use of the different data sources:
+          <b>Want to explore your data in Open Humans?
```

Comment:
I don't like this text. 

It's very targeted at existing Open Humans users and doesn't give any reason why people who aren't on Open Humans already should look at this. As we want to share this widely there should be more motivation.

With the deletions on line 9/10 there's now no idea for people to see what notebooks are etc.

Reply:
I agree that this needs to be finessed to help a user that isn't already in Open Humans.

But I'm also confused. I feel like you're saying that the jumbotron content on lines 9/10 had some link users could follow. That isn't the case. Do you think such a link should exist? Or was it something else about 9/10 that has been lost?